drivers/hyperv/hyperv.go

```diff
@@ -35,6 +41,7 @@ const (
 	defaultVLanID               = 0
 	defaultDisableDynamicMemory = false
 	defaultSwitchID             = "c08cb7b8-9b3c-408e-8e30-5e16a3aeb444"
+	defaultServerImageFilename  = "hybrid-minikube-windows-server.vhdx"
```

Comment:
is this something to be parameterized later? or does it not matter since it's only going to the cache temporarily?